### ISSUE TITLE
Upgrade to the latest sentry version

### ISSUE THREAD

**Issue by fibis** [sentry-kubernetes/charts#541]
The sentry version of this chart is pretty old. It would be great, if this could be updated.

**Comment by flowbe** [comment]
Hello, this would be really great. A lot of changes were introduced since version 21.8.0 (the one available in this chart), especially the debug symbols download from App Store Connect for iOS. We'd really need this feature but we're dependent on this repo…

**Comment by dnplkndll** [comment]
can't you just spec the image tag in the values.yml? there may be other implications of course. Chart.AppVersion is just the default. 
```
images:
  sentry:
    # repository: getsentry/sentry
    # tag: Chart.AppVersion
    # pullPolicy: IfNotPresent
    imagePullSecrets: []
```

**Comment by dnplkndll** [comment]
note on testing you do have to bump each of the image tags, here is a working latest values.yaml snip. 

```
  sentry:
    repository: getsentry/sentry
    tag: 21.12.0
#     # -> 9.1.2 -> 21.5.0 -> 21.6.3 -> latest
#     # Chart.AppVersion = 21.8.0 
#     # pullPolicy: IfNotPresent
#     imagePullSecrets: []
  snuba:
    repository: getsentry/snuba
    tag: 21.12.0
    # pullPolicy: IfNotPresent
    imagePullSecrets: []
  relay:
    repository: getsentry/relay
    tag: 21.12.0
    # pullPolicy: IfNotPresent
    imagePullSecrets: []
  symbolicator:
    repository: getsentry/symbolicator
    tag: 0.4.1
    # pullPolicy: IfNotPresent
    imagePullSecrets: []

**Comment by Mokto** [comment]
We have a major issue with Clickhouse right now: https://github.com/sentry-kubernetes/charts/pull/521

- When upgrading the performances tab does not work anymore
- On a fresh install, the migration script does not work at all.

**Comment by robjuz** [comment]
Any update on this?

**Comment by Mokto** [comment]
Fixed with release 14.0.0
